config/config.go

```diff
@@ -158,11 +160,12 @@ func NewMockConfiguration(verbosity int) Configuration {
 		UUID:     "",
 		HostName: "signalilo_appuio_lab",
 		IcingaConfig: icingaConfig{
-			URL:         "localhost:5665",
-			User:        "sepp",
-			Password:    "sepp1",
-			InsecureTLS: true,
-			Debug:       false,
+			URL:               "localhost:5665",
+			User:              "sepp",
+			Password:          "sepp1",
+			InsecureTLS:       true,
+                        DisableKeepAlives: false,
```

Comment:
Please use tabs to indent Go code. This is the standard which is applied by [gofmt](https://golang.org/cmd/gofmt/)